dbt_checkpoint/check_model_parents_schema.py

```diff
@@ -40,7 +42,13 @@ def check_parents_schema(
             )
         )
         for parent in parents:
-            db = parent.node.get("schema")
+            # Selecting the location of the model schema
+            if schema_location == "config": # pragma: no cover
```

Comment:
Why adding the `# pragma: no cover`? Please cover this case as part of the tests